app/models/accessibility_check_result.rb

```diff
@@ -3,5 +3,101 @@
 class AccessibilityCheckResult < ApplicationRecord
   belongs_to :file_resource
 
+  after_commit :broadcast_to_file_version_memberships
+
   validates :detailed_report, presence: true
+
+  def score
+    "#{num_passed} out of #{num_total} passed" if detailed_report.key?('Detailed Report')
+  end
+
+  def failures_present?
+    num_passed != num_total
```

Comment:
I noticed this bug while testing things out in the preview.  When there is an error stored in `.detailed_report`, this raises an error: https://app.bugsnag.com/penn-state/scholarsphere/errors/6725322fb73b2f0a1437c1ee?filters[event.unhandled]=true&filters[event.since]=30d&filters[app.release_stage]=acc-status .

I think we need to check `if detailed_report.key?('Detailed Report')`.